redis/cache/redisx/request_benchmark_test.go

```diff
@@ -11,6 +11,8 @@ import (
 // BenchmarkSetValue
 // BenchmarkSetValue/string
 // BenchmarkSetValue/string-12         	14946445	        67.1 ns/op
+// BenchmarkSetValue/*string
+// BenchmarkSetValue/*string-12	         6187052	        169 ns/op
```

Comment:
🔕 this looks a lot slower than others (for example, `string`). it's probably due to your machine being slower than the original source of the benchmark results, you should replace the whole result, not only add the new one you added.

Reply:
The values for others are comparable on my machine. This is slower than the `string` benchmark, because it's going from `[]byte` to `*string` so it's doing type conversion, while `string` benchmark is just doing the assignment from string to string. If I change the `string` benchmark to go from `[]byte` it's just as slow.

Reply:
I've modified the benchmarks to include source and destination types so the results are easier to interpret.